storelocator.js.org

yoriiis opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your domain request, you need to create a file named `CNAME` in the root of your github pages site (_this may not be the root of your repository_). 📝
This file should contain a single line with the full js.org domain you are requesting, in this instance `storelocator.js.org`.

yoriiis commented:
Hello @MattIPv4,

The `CNAME` file is already present
https://github.com/storelocatorjs/storelocatorjs.github.io/blob/main/docs/CNAME

I use GitHub Actions to deploy GitHub Pages, so this is linked to a branch. The last deployment job contains an artifact with the content of the website and the "CNAME" file is present
https://github.com/storelocatorjs/storelocatorjs.github.io/actions/runs/5513367345

Let me know if I need to make any other changes. Thanks!

Docs: https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow

MattIPv4 commented:
Ah, if you're using the new Actions workflow way of deploying I believe you don't need the CNAME file, you just need to set the subdomain in your settings.

MattIPv4 reviewed (approved):
✅ Commit valid, CNAME present, site appears to have content \*

\* based off https://github.com/storelocatorjs/storelocatorjs.github.io/tree/main/docs

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But as you are using a workflow to deploy the page there may be other steps involved to get the CNAME working. FYI:
https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858
https://github.com/js-org/js.org/pull/8364#issuecomment-1630895656

yoriiis commented:
@indus I changed the deployment to use a branch as a trigger and the website is available.

I have some questions:

1. Can I go back to GitHub Action mode? If I understand correctly, this manipulation was just to "activate" the site
2. Is the `https` checkbox available?
![image](https://github.com/js-org/js.org/assets/2563298/119e9d3b-d5c9-4fa4-a06d-94ea9fa48b1d)
3. I also have another domain with your extension (thanks by the way :)), I had made the request when I was using Github Pages with a branch then I changed it to use GitHub Action. Deployment works but there is a DNS warning
![image](https://github.com/js-org/js.org/assets/2563298/6bc3a5ac-672b-4cb3-96ed-5624b5d7db31)

Thanks

MattIPv4 commented:
I know other folks in the past have got stuck with the Actions workflow. Every time I have recommended reaching out to GitHub Support, as I suspect this is a breakage in this new style of deployment, yet I don't think anyone has. If you aren't able to get it to work toggling settings, I will once again recommend reaching out to support and see what they say :)

indus commented:
@yoriiis I've just checked your page and it looks ok to me: https://storelocator.js.org
Do you think it is ok now or do you wnat to change something and wait with th merge?

yoriiis commented:
@indus Yes it is ok, you can merge. Thanks for the support. I will check with the GitHub support if needed

indus commented:
Welcome https://storelocator.js.org
